src/sentry/dynamic_sampling/rules/helpers/prioritise_project.py

```diff
@@ -0,0 +1,21 @@
+from typing import TYPE_CHECKING
+
+from sentry.dynamic_sampling.rules.helpers.utils import get_redis_client_for_ds
+
+if TYPE_CHECKING:
+    from sentry.models import Project
+
+
+def _generate_cache_key(org_id, project_id) -> str:
+    return f"ds::o:{org_id}:p:{project_id}:prioritise_projects"
+
+
+def get_cached_sample_rate(project: "Project", default_samplerate=None):
```

Comment:
You might have missed the return type here, or you decided not to type hint this function?